packages/e2e-tests/tests/unencrypted-group-tests.spec.ts

```diff
@@ -17,10 +18,12 @@ const numberOfProfiles = 1
 // https://playwright.dev/docs/next/test-retries#reuse-single-page-between-tests
 let page: Page
 
-test.beforeAll(async ({ browser }) => {
+test.beforeAll(async ({ browser, chatmail }) => {
   const contextForProfileCreation = await browser.newContext()
   const pageForProfileCreation = await contextForProfileCreation.newPage()
-
+  if (chatmail) {
+    test.skip(true, 'Not suitable for non chatmail profiles')
```

Comment:
```suggestion
  if (!chatmail) {
    test.skip(true, 'Only suitable for chatmail profiles')
```

The condition is incorrect, it causes test failures.
I also simplified the log message.